.github/workflows/build_packages.yaml

```diff
@@ -427,8 +427,7 @@ jobs:
     - uses: docker/build-push-action@v2
       with:
         push: >
-          ${{ (github.event_name == 'release' && !github.event.release.prerelease)
-          || (github.event.repository.owner != 'emqx' && startsWith(github.ref_name, 'ci/')) }}
```

Comment:
Could we keep the old one ? It is convenient for me to test on forked repo,  or doe it cause issue?

Reply:
Sure, I will restore it